packages/buidler-core/test/builtin-tasks/run.ts

```diff
@@ -45,8 +45,9 @@ describe("run task", function () {
     assert.equal(process.exitCode, 0);
     (process as any).exitCode = undefined;
 
-    const files = await getAllArtifacts(path.join(process.cwd(), "artifacts"));
-    const expectedFiles = ["artifacts/contracts/a:A.json"].map((file) =>
+    const artifacts = new Artifacts(path.join(process.cwd(), "artifacts"));
+    const files = await artifacts.getArtifacts();
+    const expectedFiles = ["artifacts/contracts/a.sol/A.json"].map((file) =>
```

Comment:
I don't get why this `.map` of a 1 item literal array

Reply:
I'm not sure? Maybe I thought I was going to have other tests like this that could have multiple files. But it doesn't make sense as it is. Anyway, fixed.